### ISSUE TITLE
[20220123] Weekly AI ArXiv 만담 - 37회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#37]
- News
  - ICLR 2022 결과가 나왔습니다. 덧 ICLR 는 어떤 학회인가?
     - 올해로 10년차
     - ICML, NeurIPS가 너무 커짐에 따라 Representation learning에 집중해서 Bengio, LeCun 옹 중심으로 창설
     - 초창기 학회부터 VGG, Adam, Seq2seq with attention 등 어마무시한 연구들이 쏟아져 나옴
     - 이후 GCN, NAS, PGGAN, SNGAN, Lottery Ticket, Electra, Albert 등 한 시대를 풍미하는 연구들이 계속 쏟아져 나오고 있음.
     - 딥러닝중 Representation 중심으로 중소규모로 운영되어 (발표 200-400개 내외) 전체 프로그램 소화가 무리가 없어서 많은 ML연구자들의 최애 학회로 등극 (저도 ICLR는 2018부터 꼬박꼬박 참석)
     - 초창기부터 되게 특이한 장소에서 열리는 것으로 유명 (AI의 대중화라는 Bengio 포함 시니어 연구자들의 의지)
     - 그 정점이 ICLR 2020 에티오피아 아디스아바바 였으나 코로나로 패스... 
     - 제출하는 연구자 관점에서는 Openreview라는 신선한(?) 시스템이 매력, 
       - 익명처리 통해 모든 논문들이 제출과 동시에 공개. 그래서 리뷰어 아니어도 코멘트와 질문 달 수 있음.
       - 리뷰의 퀄리티도 상대적으로 좋고 interactive 한 discussion 도 상대적으로 활발함 (다른 학회 오버헤드의 1.5배 이상은 되는 느낌)
       - 반대로 디시전이 나오거나 withdraw하면 박제가 되고 리뷰또한 박제 되기 때문에 상대적으로 부담스러워 하는 연구자들도 있었음.
       - 그러나 제출편수가 많아짐에따라 이런 부담감도 조금씩 줄어는 상태. 
     - 특히 타 ML학회보다 diversity & inclusion, 과학문제, 세계문제에 관심이 많음.
     - 작년엔 오혜연 교수님, 올해는 최예진 교수님이 Program Chair로 활동 중
     - 이에 따라 학회 인지도도 급상승 해서 현재 Google Scholar ML 전체 1위, CS&Engineering Ranking 3위(참고로 1위는 CVPR),  전분야 10위. h5-mean 값 기준으로는 6위에 해당.
     - 그러다보니 이젠 ICLR도 너무 커짐. 발표논문 건수가 1천편 이상으로.
![image](https://user-images.githubusercontent.com/11782739/150660300-58e3d961-890d-43b6-84fd-a07e657a3280.png)
![image](https://user-images.githubusercontent.com/11782739/150660348-036d45a7-5c6e-4baf-a0db-5b269b8818a5.png)
 

  - ICASSP 취합중... (설명은 정현님이)
  - ICML 2022 데드라인이 28일 밤 9시 입니다 (27일 AOE, Supple 없음)
  - CVPR 2022 리뷰가 월 or 화에 나올 것 같습니다. --> 설명절은 저 멀리.....
- Arxiv
  - [Data2Vec](https://ai.facebook.com/blog/the-first-high-performance-self-supervised-algorithm-that-works-for-speech-vision-and-text/)
    - The first high-performance self-supervised algorithm that works for multiple modalities (?)
    - 음성, 이미지, 텍스트 인식에서 모두 뛰어난 성능이라고
    - Self-distill 처럼 student mode가 teacher mode의 layer feature representation을 복원토록 --> 그래서 modaility agnostic가능
    - 그냥하면 잘 안되서 normalization technique을 잘써야한다고 (Speech는 Instance Norm, Vision/NLP는 Layernorm)
    - 전반적으로 [iBOT](https://arxiv.org/abs/2111.07832) 과 유사한 scheme이나 hidden representatino prediction이란 차이가 있는 듯 
    - 단.. 멀티모달 모델은 아님. 동일한 아키텍처로 여러 모달리티를 학습가능한 모델(물론 feature 정의 미세 customizing 필요) 이 구조를 기반으로 멀티모달로 확장해 나가는 연구가 중요할듯
    - [논문은 여기](https://scontent-gmp1-1.xx.fbcdn.net/v/t39.8562-6/271974914_483120576492438_4239522333319653600_n.pdf?_nc_cat=107&ccb=1-5&_nc_sid=ae5e01&_nc_ohc=HLSTIdOnYI4AX9Yg9bB&_nc_ht=scontent-gmp1-1.xx&oh=00_AT_-d8rAb-bs_mbipw1Rn98DwbLGjngQ1p7-Cer9jmTDwA&oe=61F1FD91)
    - 코드는 https://github.com/pytorch/fairseq/tree/main/examples/data2vec
![image](https://user-images.githubusercontent.com/11782739/150665308-3f581b9d-3460-4d0d-bf6b-6f0a14cd16cf.png)
![image](https://user-images.githubusercontent.com/11782739/150665310-ad354607-ac24-40cf-bfbf-c715c4f0077d.png)

  - [Generating Videos with Dynamics-aware Implicit Generative Adversarial Networks](https://openreview.net/forum?id=Czsdv-S4-w9)
    - KAIST-NAVER Hypercreative AI Center 연구 from 신진우 교수님 연구실 + AI Lab
    - 기존 Video 생성의 문제는 RGB의 3d grid로 모델링 하다보니 긴 sequence 생성이 어려웠음(VideoGPT, MocoGAN-HD)
    - 최초로 Implicit neural representation을 활용하여 연속적인 Spatio / temporal coordinate 조절해서 motion dynamics를 모델링하도록 하여 Video를 생성하는 GAN (DIGAN)
    - D가 motion의 자연스러움으로 구분토록 학습하면서 더 긴 시퀀스를 학습토록, INR기반으로 속도도 훨씬 빠름
    - 128x128의 128 프레임 시퀀스 생성 가능, extrapoloation도 가능, FVD SOTA기록
    - 프로젝트 페이지 (여러가지 샘플은 여기): https://sihyun-yu.github.io/digan/
![image](https://user-images.githubusercontent.com/11782739/150672680-980f9782-e362-4114-a9a6-f1944210778a.png)
![image](https://user-images.githubusercontent.com/11782739/150672814-85f51eb0-34dc-4ed2-ad92-94ebd567921b.png)

  - [LaMDA: Language Models for Dialog Applications](https://arxiv.org/abs/2201.08239)
    - Google IO 2021 에서 공개한 LaMDA의 논문
    - 최대 137B Dialog model를 1.56T 공개 대화 및 웹텍스트 단어학습
    - Safety와 팩트 grounding (hallucination 문제) 문제해결을 위해 BlenderBot2.0과 마찬가지로 Information Retrieval System 활용하고 annotated data에 대하여 finetuning을 했음
    - SSI metric 제안 (SSA + Interestingneses by crowdworker)
![image](https://user-images.githubusercontent.com/11782739/150673235-434f41b7-46ca-4ab6-84ef-5db403d01fc1.png)
![image](https://user-images.githubusercontent.com/11782739/150673246-6e8da054-6291-42de-b1a9-19859588d6e8.png)

**Comment by ghlee0304** [comment]
- Arxiv (Audio and Speech Processing)
    - [MsEmoTTS: Multi-scale emotion transfer, prediction, and control for emotional speech synthesis](https://arxiv.org/pdf/2201.06460.pdf)
    - [Opencpop: A High-Quality Open Source Chinese Popular Song Corpus for
Singing Voice Synthesis](https://arxiv.org/pdf/2201.07429.pdf)
    - [Cross-Lingual Text-to-Speech Using Multi-Task Learning and Speaker
Classifier Joint Training](https://arxiv.org/pdf/2201.08124.pdf)

**Comment by nick-jhlee** [comment]
- News
  - AISTATS 2022 결과들도 나왔습니다!
  - ICASSP 2022 결과들도 나왔습니다!
  - COLT 2022 deadline: 02/10 아침 9시 (abstract, main, supp 모두 함께)
- Discussion
  - 과연 deep learning의 fundamental understanding에 관한 "winter"가 온 것인가...? (cf. Twitter, Tom Goldstein, Sebastian Bubeck...etc)
    - (to be continued)

**Comment by kimyoungdo0122** [comment]
- News
    - [제2회 인공지능 최고위 전략대화](https://www.etnews.com/20220119000209)
        - [인공지능 학습용 데이터, 수준 높이고 범위 넓힌다](https://www.etnews.com/20220119000067)
        - ['PIM'에 4000억원 쏟고 '데이터 수집' 허들 낮추고…'K-인공지능' 드라이브](https://www.bloter.net/newsView/blt202201180077)
        - AI 반도체,인프라 투자 및 지원 + 데이터 규제 완화 & 추가 수집 및 개방 + 중소기업 및 연구조직 혜택
![image](https://user-images.githubusercontent.com/42539153/150678767-45f2a018-304a-4ed7-9c6e-c951a93b7d91.png)
![image](https://user-images.githubusercontent.com/42539153/150679137-a68f0807-5fc6-4c5e-b721-0375d769dc83.png)


    - [IBM, 4조원 쏟고도 '의료 AI' 손 떼나…'왓슨 헬스' 매각 결정](https://www.hankyung.com/it/article/202201237252i)
        - [What Ever Happened to IBM’s Watson?](https://www.nytimes.com/2021/07/16/technology/what-happened-ibm-watson.html)
        - Today, instead of being a shorthand for technological prowess, Watson stands out as a sobering example of the pitfalls of technological hype and hubris around A.I.
        - 충분하지 않은 성능의 AI를 비즈니스에 적용하고 또 과한 투자를 하고 지나치게 마케팅한 실패한 AI 제품 사례

**Comment by hollobit** [comment]
### AI Bias에 대한 조사 보고서
https://artificialintelligence-news.com/2022/01/20/ai-bias-harms-over-a-third-of-businesses-81-want-more-regulation/
- 산업 전반에 걸쳐 350개가 넘는 조직의 응답이 포함
- AI 편향은 기업의 3분의 1 이상에 해를 끼치며 81%는 더 많은 규제를 원함
- 응답자의 절반 이상(54%)이 AI 편향의 위험에 대해 "깊은 우려"를 갖고 있고, 심지어 81%는 Bias 방지를 위해 더 많은 정부 규제를 원함 
<img width="1084" alt="스크린샷 2022-01-23 오후 10 04 22" src="https://user-images.githubusercontent.com/998803/150679839-e8c3346a-4e6d-40a6-8058-829dd04e7a48.png">

### AI 윤리 전문가 7인, 2022년 현장의 기회와 도전 전망
https://www.morningbrew.com/emerging-tech/stories/2022/01/17/seven-ai-ethics-experts-predict-2022-s-opportunities-and-challenges-for-the-field
- 제기한 질문: "올해 AI 윤리 분야에서 가장 큰 발전이 예상되는 것은 무엇입니까? 반대로 가장 중요한 과제는 무엇입니까?”
- 사용자를 위한 알고리즘 선택의 개선 등등

### Google 연구: 2021년 이후의 주제
https://ai.googleblog.com/2022/01/google-research-themes-from-2021-and.html
https://analyticsindiamag.com/google-ais-plan-for-2022-and-beyond/
- Google Research의 선임 연구원이자 SVP인 Jeff Dean
 * 트렌드 1: 더 많은 기능을 갖춘 범용 ML 모델
 * 트렌드 2: ML의 지속적인 효율성 개선
 * 트렌드 3: ML은 점점 더 개인적으로나 공동체적으로 유익해지고 있습니다.
 * 트랜드 4: 과학, 건강 및 지속 가능성에서 ML의 이점 증가
 * 트렌드 5: ML에 대한 더 깊고 폭넓은 이해

**Comment by veritas9872** [comment]
PyTorch 프로젝트에서 Reproducibility 및 CUDA environment setup 이슈를 모두 해결해주는 Cresset Project 다시 공유해드립니다. 
여러 Performance enhancement 뿐만 아니라 UI와 UX를 인공지능 연구원 및 개발자를 위해 최적화했습니다.
여러분의 성원에 힘입어 star 560를 달성해서 version 0.2를 release했습니다. 많은 사용 부탁드립니다!

**Comment by veritas9872** [comment]
조금 전에 링크 추가하는 것을 잊어버렸네요 ^^;;
